### DISCUSSION TITLE
Is NomNom Dead?

### DISCUSSION THREAD

**Discussion by jerj** [zencq/NomNom#161] [General]
The latest beta is quite broken and there doesn't appear to be any progress for quite some time.

**Comment by earwigcahootz** [comment]
I'm late to the NomNom party so I'm wondering as well. I found it from the Assistant app but can't get anything to work. I installed the recent version (Win11), and it finds my saves, but clicking any save and then "Edit Save" does nothing. If I click the Assistant for No Man's Sky button the app crashes.

**Comment by cengelha** [comment]
To make this "official" here: No, it is not dead. I know some things are somewhat broken and I'm working on a bigger update in the background and investing more time in that than the current one.
Besides that, things should have been improved since this was written (with the beta.7 release).
